microstream-cache/src/main/java/io/micronaut/microstream/cache/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Microstream classes related to caching.
+ *
+ * @see <a href="https://micronaut-projects.github.io/micronaut-cache/latest/guide/">Micronaut Cache</a>
+ * @author Tim Yates
+ * @since 1.0.0
+ */
+package io.micronaut.microstream.cache;
```

Comment:
In micronaut-cache we add `@Require` statements here:

https://github.com/micronaut-projects/micronaut-cache/blob/master/cache-infinispan/src/main/java/io/micronaut/cache/infinispan/package-info.java#L22-L23

Reply:
resolved via https://github.com/micronaut-projects/micronaut-microstream/pull/23/commits/71624f8def506a8393f74286dcdf9711c94c86cd